logstash-core/lib/logstash/plugins/mixin.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+
+module LogStash
+  module Plugins
+    module Mixin
+
+      def annotate(name="")
+        if !name.empty?
+          super("filter", name)
```

Comment:
Good catch, for now I left this like this because I wanted to see if you liked the procedure. Easy to extend to others following this idea.